build/configs/rtl8720e/board_metadata.txt

```diff
@@ -0,0 +1,28 @@
+###########################################################################
+#
+# Copyright 2022 Samsung Electronics All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+# either express or implied. See the License for the specific
+# language governing permissions and limitations under the License.
+#
+###########################################################################
+# board_metadata.txt
+
+BOARD="RTL8720E"
+
+DEFAULT_PORT="ttyUSB1"
+
+BL1=""
+BL2=""
+KERNEL=""
```

Comment:
Could you add the output name like km0_km4_image2.bin?

Reply:
The output for 8720e is actually kr4_km4_image2.bin, and yes, I will add it and others like 8721csm